atlasdb-impl-shared/src/test/resources/lockwatch-event-cache-output/getCommitUpdateIsInvalidateSomeAsEventsAreRetained/event-cache-1.json

```diff
@@ -0,0 +1,82 @@
+{
+  "logState" : {
+    "snapshotState" : {
+      "watches" : [ ],
+      "locked" : [ {
+        "bytes" : "dGFibGUAAg=="
+      } ],
+      "snapshotVersion" : {
+        "id" : "470c855e-f77b-44df-b56a-14d3df085dbc",
+        "version" : 3
+      }
+    },
+    "eventStoreState" : {
+      "eventMap" : {
+        "4" : {
+          "type" : "created",
+          "sequence" : 4,
+          "references" : [ {
+            "type" : "fullTable",
+            "qualifiedTableRef" : "table"
+          } ],
+          "lockDescriptors" : [ {
+            "bytes" : "dGFibGUAAQ=="
+          } ]
+        },
+        "5" : {
+          "type" : "unlock",
+          "sequence" : 5,
+          "lockDescriptors" : [ {
+            "bytes" : "dGFibGUAAg=="
+          } ]
+        },
+        "6" : {
+          "type" : "lock",
+          "sequence" : 6,
+          "lockDescriptors" : [ {
+            "bytes" : "dGFibGUAAw=="
+          } ],
+          "lockToken" : {
+            "requestId" : "203fcd7a-b3d7-4c2a-9d2c-3d61cde1ba59"
+          }
+        },
+        "7" : {
+          "type" : "lock",
+          "sequence" : 7,
+          "lockDescriptors" : [ {
+            "bytes" : "dGFibGUAAQ=="
+          } ],
+          "lockToken" : {
+            "requestId" : "888fcd7a-b3d7-4d2a-9d2c-3d61cde1ba44"
+          }
+        }
+      }
+    },
+    "latestVersion" : {
+      "id" : "470c855e-f77b-44df-b56a-14d3df085dbc",
+      "version" : 7
+    }
+  },
+  "timestampStoreState" : {
+    "timestampMap" : {
+      "1" : {
+        "version" : {
+          "id" : "470c855e-f77b-44df-b56a-14d3df085dbc",
+          "version" : 3
+        },
+        "commitInfo" : {
+          "commitLockToken" : {
+            "requestId" : "203fcd7a-b3d7-4c2a-9d2c-3d61cde1ba59"
+          },
+          "commitVersion" : {
+            "id" : "470c855e-f77b-44df-b56a-14d3df085dbc",
+            "version" : 6
+          }
+        }
+      }
+    },
+    "livingVersions" : {
+      "3" : [ 1 ]
```

Comment:
This is not intuitive since we processed the commit timestamps update, but I assume that is because it is living until snapshot transaction manager tells the LWEC to remove it?